doc/changelog.md

```diff
@@ -539,6 +637,9 @@
 * Catalyst's implementation of Lightning Kokkos plugin has been removed in favor of Lightning's one.
   [(#974)](https://github.com/PennyLaneAI/catalyst/pull/974)
 
+* Eliminate (some) scalar tensors from the IR by adding a `linalg-detensorize` pass at the end of the HLO lowering passes.
```

Comment:
```suggestion
* Eliminate some scalar tensors from the IR by adding a `linalg-detensorize` pass at the end of the HLO lowering pipeline.
  This will reduce memory usage at runtime as well reduce compilation complexity.
```